src/Symfony/Component/Cache/Tests/Adapter/FilesystemAdapterTest.php

```diff
@@ -49,4 +49,62 @@ public static function rmdir($dir)
         }
         rmdir($dir);
     }
+
+    public function testPrune()
```

Comment:
let's go with duplication between Adapter and Simple, which may allow both to diverge (eg populating the pool can then be done using each's interfaces)